core/src/main/java/dk/alexandra/fresco/framework/util/HmacDrbg.java

```diff
@@ -0,0 +1,143 @@
+package dk.alexandra.fresco.framework.util;
+
+import dk.alexandra.fresco.framework.MPCException;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.function.Supplier;
+import javax.crypto.Mac;
+import javax.crypto.SecretKey;
+import javax.crypto.spec.SecretKeySpec;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This class implements the HMac-DRBG (HMac-Deterministic Random Bit Generator) as specified in
+ * https://csrc.nist.gov/publications/detail/sp/800-90a/rev-1/final
+ *
+ * <p>
+ * Note that this class is not threadsafe.
+ * </p>
+ * <p>
+ * Note also that reseeding (calling {@link #setSeed(byte[])}) should occur at least every 2^42
```

Comment:
This cannot be done as setSeed is private. This seems like  outdated documentation.